pallets/nfts/src/lib.rs

```diff
@@ -73,9 +76,13 @@ pub mod pallet {
         /// generated and logged as an event, The caller of this function
         /// will become the owner of the new NFT.
         #[pallet::weight(T::WeightInfo::create())]
-        pub fn create(origin: OriginFor<T>, details: T::NFTDetails) -> DispatchResultWithPostInfo {
+        pub fn create(
+            origin: OriginFor<T>,
+            details: T::NFTDetails,
+            series_id: T::NFTSeriesId,
+        ) -> DispatchResultWithPostInfo {
             let who = ensure_signed(origin)?;
-            let _id = <Self as NFTs>::create(&who, details)?;
+            let _id = <Self as NFTs>::create(&who, details, series_id)?;
```

Comment:
We need to check wether `who` owns the serie or not

Reply:
It is now checked :)